MSAL/src/instance/MSALAuthority.h

```diff
@@ -60,7 +60,7 @@ typedef void(^MSALAuthorityCompletion)(MSALAuthority *authority, NSError *error)
 
 @property MSALAuthorityType authorityType;
 @property NSURL *canonicalAuthority;
-@property BOOL validateAuthority;
+@property BOOL validatedAuthority;
```

Comment:
nit: if this is meant to represent an authority that's been validated, then maybe "isValidatedAuthority" would be clearer?